.prettierignore

```diff
@@ -2,3 +2,4 @@ dist/
 node_modules/
 CHANGELOG.md
 package-lock.json
+/src/components/Modal.astro
```

Comment:
I added this here because I kept running to prettier parsing errors when trying to push my changes up.
There was an issue parsing my script section in the Modal.astro file even though I couldn't find any formatting errors.
I am not sure how to resolve this.

Reply:
If there's part of the file that is causing issues with prettier, you can have prettier ignore it. See https://prettier.io/docs/en/ignore.html. That way we can still have prettier formatter the other parts of the file.

Reply:
Since we are moving to a react component then this issue should resolve itself since the issues was around the script element inside the astro file specifically. 

I will go ahead and remove this change in the ignore file.